project-fortis-interfaces/src/components/Admin/UserRoles.js

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
+import Snackbar from 'material-ui/Snackbar';
+import groupBy from 'lodash/groupBy';
+import differenceBy from 'lodash/differenceBy';
+import { DataGrid } from './DataGrid';
+import { getColumns } from './shared';
+const { Editors, Formatters } = require('react-data-grid-addons');
+const { DropDownEditor } = Editors;
+const { DropDownFormatter } = Formatters;
+
+const roles = ['user', 'admin'];
+
+class UserRoles extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      open: false,
+      message: ''
+    };
+
+    this.getUserRolesColumns = this.getUserRolesColumns.bind(this);
```

Comment:
NB: you can avoid to use `bind` everywhere by using class-level methods which also preserve the `this`-context. Means less typing and less chances of forgetting to bind and getting weird bugs.

```js
getUserRolesColumns = () => {
 // ... implementation here
}
```

Background reading: section five [in this article](https://medium.freecodecamp.org/react-binding-patterns-5-approaches-for-handling-this-92c651b5af56).